docs/cristin.ttl

```diff
@@ -103,22 +112,23 @@
          """ .
 
 :FAGFELT_Class
-    a       :CristinClass;
-    rdfs:comment """The scientific field in which the book is published""" ;
+    a               :CristinClass ;
+    rdfs:comment    """The scientific field in which the book is published""" ;
     rdfs:subClassOf :TYPE_BOK_RAPPORT_Class .
 
 :TYPE_TIDSSKRIFTPUBLIKASJON_Class
-    a       :CristinClass;
-    rdfs:comment """A type publication called tidsskrift""".
+    a            :CristinClass ;
+    rdfs:comment """A type publication called tidsskrift""" .
 
 :TIDSSKRIFT_Class
-    a       :CristinClass;
+    a            :CristinClass ;
     rdfs:comment """UNKNOWN""" .
```

Comment:
Contains information about a journal where a result i published.

NB: For e.g. books that are published as part of a series, the series is also modeled as a TIDSSKRIFT (called TIDSSKRIFT_SERIE in the JSON export, but containing the same fields and stored in the same DB table). I guess this is because both have a name and an ISSN, so they were modeled together.

The JSON at least gives them two different field names, so there's that.